test/TestDSP.c

```diff
@@ -102,6 +112,19 @@ void test_dsp_object(void)
     TEST_ASSERT_DOUBLE_WITHIN(0.01, -16.653, dspv5_object_helper(-5000, -5000, 22454, 23030));
 }
 
+void test_dsp_object_reflected(void)
+{
+    double ke = mlx90632_get_emissivity();
+
+    mlx90632_set_emissivity(1.0);
+    TEST_ASSERT_DOUBLE_WITHIN(0.01, dspv5_helper(609, 611, 22454, 23030), dspv5_object_reflected_helper(609, 611, 22454, 23030, 40.00));
```

Comment:
`dspv5_helper` i think here should be something else? I think you wanted `dspv5_object_helper` probably to show it is same for both when emissivity is 1?